Source/SpectrumAnalysis/KTSeqTrackFinder.cc

```diff
@@ -0,0 +1,539 @@
+/*
+ * KTSeqTrackFinder.cc
+ *
+ *  Created on: Sep 15, 2016
+ *      Author: Christine
+ */
+
+#include "KTSeqTrackFinder.hh"
+#include "KTLogger.hh"
+
+
+#include <numeric>
+#include <cmath>
+
+using std::list;
+using std::set;
+using std::vector;
+
+
+namespace Katydid
+{
+    KTLOGGER(stflog, "DansTrackFinder");
```

Comment:
Let's make the official name of this algorithm the Sequential Track Finder, unless you have another name you'd like to use.  Please rename the processor files KTSequentialTrackFinder.hh/cc, and in the KTLOGGER line here, you can say `SequentialTrackFinder`.